src/fontra/client/web-components/designspace-location.js

```diff
@@ -4,7 +4,7 @@ import * as html from "/core/html-utils.js";
 import { UnlitElement, htmlToElement } from "/core/html-utils.js";
 
 const colors = {
-  "disabled-color": ["#ccc", "#777"],
+  "disabled-text-color": ["#777", "#ccc"],
```

Comment:
Please undo the changes in this file. The text request was specifically about the numeric input, not the info box.

Reply:
But... please change the `gap` of `.slider-group` to `0.1em`

Reply:
Done.